### PR TITLE
Add script for deploying Simple PRE

### LINKED ISSUE #40: Ability to deploy PRE Application Contract

### PR DESCRIPTION
**Pull request by theref**
Closes #40

### CODE REVIEW

**Review by cygnusv — APPROVED** [review]

File: scripts/deploy_simple_pre.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+def main(account_id=None):
+    deployer = get_account(account_id)
+    network_config = config["networks"][CURRENT_NETWORK]
+
+    if CURRENT_NETWORK in LOCAL_BLOCKCHAIN_ENVIRONMENTS:
+        _, _, t_staking, _, _ = deploy_mocks(deployer)
+    else:
+        t_staking = network_config.get("t_staking")
+
+    adjudicator = SimplePREApplication.deploy(
```
[the remark this answers is not in the document]
Comment (theref) [reply scripts/deploy_simple_pre.py]: fixed

File: brownie-config.yaml [hunk 2]
```diff
@@ -17,12 +17,20 @@ networks:
...
     work_lock: '0xe9778E69a961e64d3cdBB34CF6778281d34667c2'
+    staking_escrow: '0xbbD3C0C794F40c4f993B03F65343aCC6fcfCb2e2'
+    pre_min_authorization: 40000000000000000000000
+    pre_min_operator_seconds: 86400  # one day in seconds
     verify: True
   development:
     nu_token_supply: 1_000_000_000
+    pre_min_authorization: 40000000000000000000000
+    pre_min_operator_seconds: 86400  # one day in seconds
     verify: False
```
Comment (vzotova) [inline brownie-config.yaml:36]:
```suggestion
    verify: False

```

File: scripts/deploy_adjudicator.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python3
+from brownie import Adjudicator, config
+from scripts.utils import CURRENT_NETWORK, LOCAL_BLOCKCHAIN_ENVIRONMENTS, deploy_mocks, get_account
+
+
+def main(account_id=None):
```
Comment (vzotova) [inline scripts/deploy_adjudicator.py:6]: I guess this file should be removed?

File: brownie-config.yaml (hunk 2 above)
Comment (theref) [reply brownie-config.yaml:36]: why remove these? they are required by the constructor, no?

**Pull request merged**